{
  "metadata": {
    "timestamp": "2025-02-23T20:11:11Z"
  },
  "data": [
    {
      "cve": {
        "id": "CVE-2020-24741",
        "description": "Rejected reason: DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: CVE-2020-0570. Reason: This candidate is a duplicate of CVE-2020-0570. A typo caused the wrong ID to be used. Notes: All CVE users should reference CVE-2020-0570 instead of this candidate. All references and descriptions in this candidate have been removed to prevent accidental usage",
        "metadata": {
          "published_date": "2021-08-09T22:15:08.563",
          "last_modified_date": "2023-11-07T03:20:10.170",
          "confidence_level": "low",
          "severity": "n/a"
        },
        "impact": {
          "cisa_kev": false,
          "reported_exploited": false,
          "exploit_maturity": "none",
          "automatable": false
        },
        "counts": {
          "public_exploit_count": 0
        },
        "epss": {
          "score": 0.00241,
          "percentile": 0.63763
        },
        "metrics": [],
        "timeline": {
          "references": [],
          "repositories": []
        }
      }
    }
  ]
}